visualization/app/codeCharta/util/customConfigHelper.ts

```diff
@@ -155,7 +154,8 @@ export class CustomConfigHelper {
 				customConfigVersion: exportedConfig.customConfigVersion,
 				mapChecksum: exportedConfig.mapChecksum,
 				mapSelectionMode: exportedConfig.mapSelectionMode,
-				stateSettings: exportedConfig.stateSettings
+				stateSettings: exportedConfig.stateSettings,
+				camera: exportedConfig.camera
```

Comment:
exportedConfig.camera is not available in old configs that has been created based on current main.

Reply:
same as [this comment](https://github.com/MaibornWolff/codecharta/pull/2880#discussion_r919175014)